PKHeX/Legality/Tables4.cs

```diff
@@ -570,9 +570,10 @@ public static partial class Legal
         #endregion
 
         // Encounter Slots that are replaced
+        internal static readonly int[] Slot4_Swarm = {0, 1};
         internal static readonly int[] Slot4_Time = {2, 3};
         internal static readonly int[] Slot4_Sound = {2, 3, 4, 5};
-        internal static readonly int[] Slot4_Radar = {6, 7, 10, 11};
+        internal static readonly int[] Slot4_Radar = {4, 5, 10, 11};
```

Comment:
But when I read from the data, I can see when there is no pokeradar encounter, the species ID are identical to slots 4,5. Also if I change slots to 4,5 the save file of RoC will be all clear (legal) now